docroot/modules/custom/layout_builder_custom/layout_builder_custom.module

```diff
@@ -141,6 +143,10 @@ function layout_builder_custom_form_alter(&$form, FormStateInterface $form_state
       unset($form['settings']['label']);
       unset($form['settings']['label_display']);
       unset($form['settings']['admin_label']['#title']);
+      // Hide ones implemented by Views because ListBlock has headline.
+      unset($form["settings"]["views_label_fieldset"]);
+      unset($form["settings"]["views_label_checkbox"]);
+      unset($form["settings"]["views_label"]);
```

Comment:
I can move it. I think I did that first but went this direction because of the similar overrides above it. Yeah I can switch to single quotes. Need to figure out how to change xDebug to copy path using single quotes.

Reply:
Changed to single quotes. Can't seem to target these "settings" in blockForm. Added a todo to investigate at a later date.